tests/queries/0_stateless/02972_insert_deduplication_token_hierarchical_inserts.sql

```diff
@@ -101,3 +101,54 @@ DROP VIEW mv_2_2;
 
 DROP TABLE ds_3_1;
 DROP VIEW mv_3_1;
+
```

Comment:
We need to extract this to separate tests.
Maybe `insert_deduplication_token_hierarchical_inserts_views.sql`.